src/format/KdbxXmlReader.cpp

```diff
@@ -814,26 +814,14 @@ void KdbxXmlReader::parseEntryString(Entry* entry)
             QXmlStreamAttributes attr = m_xml.attributes();
             value = readString();
 
+            if (m_error) {
+                raiseError(tr("Unable to decrypt entry string"));
```

Comment:
This discards the randomStream error.

Reply:
That's intended to preserve the previous behavior and return another message in this case. Should I just remove this? In that case, also remove the translations?

Reply:
I think it makes more sense to return the actual stream error message.